modules/ROOT/pages/_partials/data-sync/server-realtime-updates.adoc

```diff
@@ -53,15 +53,16 @@ const apolloServer = VoyagerServer({
 
 const app = express()
 apolloServer.applyMiddleware({ app })
+const port = 4000;
```

Comment:
```suggestion
const port = 4000
```

I definitely don't want to get into a flamewar around semicolons vs. no semicolons, however there aren't any being used within this code snippet so we should at least keep it consistent on that level 😃

Reply:
@darahayes  that seems fair, are you ok with the edit from @MStokluska  ?